src/app/rpmostree-builtin-deploy.c

```diff
@@ -40,7 +40,7 @@ static GOptionEntry option_entries[] = {
    *     A --preview option would work for both commands if we wanted to
    *     deprecate --check-diff. */
   { "preview", 0, 0, G_OPTION_ARG_NONE, &opt_preview, "Just preview package differences", NULL },
-  { "cache-only", 'C', 0, G_OPTION_ARG_NONE, &opt_cache_only, "Do not update repo metadata cache", NULL },
+  { "cache-only", 'C', 0, G_OPTION_ARG_NONE, &opt_cache_only, "Do not download latest ostree and rpmmd data", NULL },
```

Comment:
"rpmmd data" is a bit redundant (though I personally find myself saying "ATM machine" too...).  Maybe "rpm metadata"?

Reply:
I changed it to `RPM data` since it's not just metadata anymore, but also the RPM themselves.